src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{

```

Comment:
There is a missing space between the class name 'Machine' and the opening brace '{'. It should be `extends Machine {` to follow Java code conventions.